### ISSUE TITLE
[Customer Query] DOM Element warnings on activity page

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtMediaWP/rtMedia#1565]
DOM element warnings are shown on activity page.

![Screenshot from 2020-03-02 13-37-29](https://user-images.githubusercontent.com/41000648/75656904-0b6b1c80-5c8b-11ea-89c9-1befed3d8037.png)

----

Update this customer once release it out - https://wordpress.org/support/topic/non-unique-id-rt_upload_hf_comment/

**Comment by SID177** [comment]
![Screenshot 2020-03-02 at 5 15 55 PM](https://user-images.githubusercontent.com/23107546/75673704-90fdc500-5ca9-11ea-8b5c-267cf145beb0.png)
These warnings (`_wpnonce_new_activity_comment`) are coming from buddypress so can't do anything about these.

**Comment by SID177** [comment]
We can fix warnings related duplicate Ids - `rt_upload_hf_comment, rt_upload_hf_privacy, rt_upload_hf_upload_parent_id_context and rtmedia_upload_nonce`

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested with `pre-release` branch and the fix added works, the warnings with `rt_upload_hf_comment` are been removed from activity page. Checked in both profile activity and group activity.

The warnings `#_wpnonce_new_activity_comment` are related to BuddyPress  and need to be fixed in BuddyPress plugin. https://github.com/rtMediaWP/rtMedia/issues/1565#issuecomment-593364149

```
Test Environment
WordPress: v5.4-RC-2
rtMedia: v4.6.0  pre-release branch
BuddyPress: v5.1.2 Legacy and Nouveau
Theme: 2020 2019

**Comment by kjnanda** [comment]
@SID177 Please check my comment below.

1. The added fix removes the warning from the console. ✅ 
2. When the sidebar widget is active, it still shows some warnings. We need to fix this part. 

Please check this screenshot.

![Fullscreen_3_19_20__3_09_PM](https://user-images.githubusercontent.com/15173772/77053177-f238ce80-69f3-11ea-8ca8-4689f90edeec.png)

The highlighted part shown for `rtmedia upload terms` in the screenshot, appears when the sidebar widget add-on is active. If I deactivate the add-on but upload terms are still enabled then it would not show the same warning.

**Comment by SID177** [comment]
The warnings related to `rtMedia-start-upload-rtmediauploaderwidget_2` will be fixed in rtMedia Sidebar Widget addon

**Comment by kjnanda** [comment]
@SID177 Added changes LGTM. For subtask - 3, I am creating an issue in add-on itself.

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V4.6.1
